chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SearchViewController.java

```diff
@@ -393,4 +400,20 @@ public void download(@RequestParam(value="edition", required=false) String editi
 		return searchMenuManager.getPopulateSearchOptions(simple);
 	}
 	
+	@ApiOperation(value="Get all developer decertifications in the CHPL", 

```

Comment:
I'd be wary of putting it in the DeveloperController, knowing we're going to be doing Decertified Certified Products next. I think those Decertified things should stay together, whether in the Search or in a different Controller